lib/modules/manager/npm/update/dependency/index.ts

```diff
@@ -129,8 +137,20 @@ export function updateDependency({
   newValue = getNewNpmAliasValue(newValue, upgrade) ?? newValue;
 
   logger.debug(`npm.updateDependency(): ${depType}.${depName} = ${newValue}`);
+
+  let isYaml = false;
```

Comment:
Can we instead update the `updateDependency()` function to include an `isYaml` parameter so we don't have to try/catch/try/catch?